### PR TITLE
TOTP: Remove 'View Options' when totp is configured.

### PR DESCRIPTION
**Pull request by joshbetz**
See #158

### CODE REVIEW

File: providers/class.two-factor-totp.php [hunk 1]
```diff
@@ -74,23 +74,24 @@ public function user_two_factor_options( $user ) {
...
-				<p><?php esc_html_e( 'Please scan the QR code or manually enter the key, then enter an authentication code from your app in order to complete setup', 'two-factor' ); ?></p>
-				<p>
-					<label for="two-factor-totp-authcode"><?php esc_html_e( 'Authentication Code:', 'two-factor' ); ?></label>
-					<input type="hidden" name="two-factor-totp-key" value="<?php echo esc_attr( $key ) ?>" />
-					<input type="tel" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9]*" />
-				</p>
-			<?php } else { ?>
-				<p class="success"><?php esc_html_e( 'Enabled', 'two-factor' ); ?></p>
-			<?php } ?>
+		<?php if ( false && empty( $key ) ):
```
Comment (kasparsd) [inline providers/class.two-factor-totp.php]: @joshbetz The `false` is probably a leftover from debugging, right?

File: providers/class.two-factor-totp.php [hunk 2]
```diff
@@ -74,23 +74,24 @@ public function user_two_factor_options( $user ) {
...
+					<p><strong><?php echo esc_html( $key ); ?></strong></p>
+					<p><?php esc_html_e( 'Please scan the QR code or manually enter the key, then enter an authentication code from your app in order to complete setup', 'two-factor' ); ?></p>
+					<p>
+						<label for="two-factor-totp-authcode"><?php esc_html_e( 'Authentication Code:', 'two-factor' ); ?></label>
+						<input type="hidden" name="two-factor-totp-key" value="<?php echo esc_attr( $key ) ?>" />
+						<input type="tel" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9]*" />
+					</p>
+			</div>
+		<?php else: ?>
+			<p class="success"><?php esc_html_e( 'Enabled', 'two-factor' ); ?></p>
```
Comment (kasparsd) [inline providers/class.two-factor-totp.php]: @joshbetz This "success" message is currently inside the `empty( $key )` conditional. It should be visible when the user has actually configured TOTP.

Comment (joshbetz) [inline providers/class.two-factor-totp.php]: This is in the `else` on line 92.

Comment (kasparsd) [inline providers/class.two-factor-totp.php]: Haha, I guess it's too late for me to do code reviews :) Sorry about that.

File: providers/class.two-factor-totp.php [hunk 3]
```diff
@@ -74,23 +74,24 @@ public function user_two_factor_options( $user ) {
...
-				<p><?php esc_html_e( 'Please scan the QR code or manually enter the key, then enter an authentication code from your app in order to complete setup', 'two-factor' ); ?></p>
-				<p>
-					<label for="two-factor-totp-authcode"><?php esc_html_e( 'Authentication Code:', 'two-factor' ); ?></label>
-					<input type="hidden" name="two-factor-totp-key" value="<?php echo esc_attr( $key ) ?>" />
-					<input type="tel" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9]*" />
-				</p>
-			<?php } else { ?>
-				<p class="success"><?php esc_html_e( 'Enabled', 'two-factor' ); ?></p>
-			<?php } ?>
+		<?php if ( empty( $key ) ):
```
Comment (kasparsd) [inline providers/class.two-factor-totp.php]: @joshbetz Just this last one:

```
## PHP_CodeSniffer
providers/class.two-factor-totp.php:77:34: error - Expected 1 space after closing parenthesis; found 0 (Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis)
providers/class.two-factor-totp.php:77:35: error - Space between opening control structure and T_COLON is required (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBetweenStructureColon)
```

**Comment by kasparsd** [comment]
@georgestephanis Could we merge this in?

**Pull request merged**